mypyc/irbuild/specialize.py

```diff
@@ -710,6 +711,28 @@ def translate_fstring(builder: IRBuilder, expr: CallExpr, callee: RefExpr) -> Va
                 format_ops.append(FormatOp.STR)
                 exprs.append(item.args[0])
 
+        for i in range(len(exprs) - 1):
+
+            def get_literal_str(expr: Expression) -> str | None:
+                # NOTE: not sure where I should put this helper, this file? somewhere else?
```

Comment:
It seems fine to keep it here. Similar functionality is already used in a few other files, but all of the use cases look slightly different so sharing doesn't look essential. You can remove this comment.